framework/include/auxkernels/ADMaterialStdVectorAux.h

```diff
@@ -0,0 +1,36 @@
+//* This file is part of the MOOSE framework
```

Comment:
Can you template the existing class instead of making a new one?

Reply:
Yes, getting there :)